api/swagger.yaml

```diff
@@ -6277,6 +6277,7 @@ paths:
             - `name=<network-name>` Matches all or part of a network name.
             - `scope=["swarm"|"global"|"local"]` Filters networks by scope (`swarm`, `global`, or `local`).
             - `type=["custom"|"builtin"]` Filters networks by type. The `custom` keyword returns all user-defined networks.
+            - `dangling=["true"|"false"] Print only dangling networks`
```

Comment:
will do s/Print/Prints. 

About '0' and '1', I'm a little confused.

```
~ docker image ls --filter dangling=0
Error response from daemon: Invalid filter 'dangling=[0]'

~ docker volume ls --filter dangling=0
DRIVER              VOLUME NAME
```
So I'm on the fence with that